lib/options.js

```diff
@@ -279,6 +276,16 @@ module.exports = {
         external: true,
         usage: ['CONVERSION'],
         supportedIn: [VERSION30, VERSION31]
+      },
+      {
+        name: 'Include webhooks',
+        id: 'includeWebhooks',
+        type: 'boolean',
+        default: false,
+        description: 'Add/Remove webhooks folder to the collection',
```

Comment:
maybe we could change it to 'Select whether to include Webhooks in the generated collection', because we are not removing anything really